src/components/CurrentLocation.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class CurrentLocation extends React.Component {
+
+  render() {
+    const { location } = this.props;
+    const circleStyle = {
+      background: location.color
+    };
```

Comment:
you'll do better to use external CSS and classNames for styling and write this as a pure function of its props